addons/l10n_es_edi_tbai/wizards/account_move_send.py

```diff
@@ -0,0 +1,99 @@
+from odoo import _, api, fields, models
+
+
+class AccountMoveSend(models.TransientModel):
+    _inherit = 'account.move.send'
+
+    l10n_es_tbai_enable_send = fields.Boolean(compute='_compute_l10n_es_tbai_enable_send')
```

Comment:
```suggestion
    l10n_es_tbai_show_checkbox = fields.Boolean(compute='_compute_l10n_es_tbai_show_checkbox')
```
To avoid confusion.

Reply:
It would not be a bad idea as well to disable to checkbox when the Ticketbai has been sent already.

Reply:
Should we even allow to send an invoice multiple times to TBAI?